pkg/parse/parse.go

```diff
@@ -612,19 +627,39 @@ func PlatformFromOptions(c *cobra.Command) (os, arch string, err error) {
 			arch = selectedArch
 		}
 	}
-
+	platforms = []struct{ OS, Arch, Variant string }{{os, arch, ""}}
+	platformSpecs := ""
 	if c.Flag("platform").Changed {
-		if pf, err := c.Flags().GetString("platform"); err == nil {
-			selectedOS, selectedArch, _, err := Platform(pf)
+		platformSpecs, err = c.Flags().GetString("platform")
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to parse platform")
+		}
+	}
+	if c.Flag("platforms").Changed {
+		platformSpecs, err = c.Flags().GetString("platforms")
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to parse platforms")
+		}
+	}
+	if platformSpecs != "" {
+		for _, pf := range strings.Split(platformSpecs, ",") {
+			selectedOS, selectedArch, selectedVariant, err := Platform(pf)
 			if err != nil {
-				return "", "", errors.Wrap(err, "unable to parse platform")
+				return nil, errors.Wrapf(err, "unable to parse platform %q", pf)
 			}
-			arch = selectedArch
-			os = selectedOS
+			platforms = append(platforms, struct{ OS, Arch, Variant string }{selectedOS, selectedArch, selectedVariant})
 		}
 	}
-
-	return os, arch, nil
+	add := true
+	for _, pf := range platforms {
+		if os == "" || pf.OS == os && arch == "" || pf.Arch == arch {
+			add = false
+		}
+	}
+	if add {
+		platforms = append([]struct{ OS, Arch, Variant string }{{os, arch, ""}}, platforms...)
```

Comment:
Is casing a concern?

Reply:
Can you elaborate on that?  I don't think I follow the question.

Reply:
Just wondering if someone does Fedora/arm, fedora/ARM, FEDORA/Arm, or what have you, if the system will handle the casing of the values to the right case, or will it just spit them out?

Reply:
OS and architecture names are case-sensitive, yes.